tests/meta/workflow/conftest.py

```diff
@@ -158,20 +193,35 @@ def qc_microsalt_context(
         internal_id=microsalt_case_qc_pass,
         name=microsalt_case_qc_pass,
         data_analysis=Pipeline.MICROSALT,
+        action=CaseActions.RUNNING,
     )
 
-    for sample in qc_pass_microsalt_samples:
+    for sample in qc_pass_microsalt_samples[1:]:
         sample_to_add: Sample = helpers.add_sample(
             store=store,
             internal_id=sample,
             application_tag=MicrosaltAppTags.MWRNXTR003,
             application_type=MicrosaltAppTags.PREP_CATEGORY,
-            reads=MicrosaltQC.TARGET_READS,
+            reads=MicrosaltQC.TARGET_READS * 2,
```

Comment:
Is this added because we require strictly more reads?

Reply:
No, the requirement is some percentage of the target reads. But this ensures all the samples pass the threshold